server.go

```diff
@@ -185,6 +186,9 @@ func (s Server) DataToMap() map[string]interface{} {
 }
 
 func parseRequirementRow(str string) []string {
+	if str == "" {
```

Comment:
Return an empty array instead, nil is special, where-as empty is not.

Reply:
Done. (I haven't written much Go and wasn't sure if empty vs. nil slice was preferred.)